resources/profiles/Creality.ini

```diff
@@ -64,9 +64,9 @@ max_print_speed = 100
 max_volumetric_extrusion_rate_slope_negative = 0
 max_volumetric_extrusion_rate_slope_positive = 0
 max_volumetric_speed = 0
-min_skirt_length = 4
+min_skirt_length = 15
```

Comment:
A skirt length of 15 seems excessive and wasteful of filament for no added benefit. Generally, skirts are at 2-4 lines to prime the extruder. Brims are generally at 15-30 lines for bed adhesion. 
(just reviewing changes to see what might apply to my printer's config bundle which was heavily influenced by the included creality profiles, Saw this and it seems like an odd change).